src/patch.js

```diff
@@ -58,16 +58,18 @@ function setElementProp(element, name, value, oldValue) {
       element.style[name] = value[name] != null ? value[name] : ""
```

Comment:
Not part of this PR, but could you change this to `element[name][i]` and also the loop variable from `name` to `i`?

Reply:
Oh, I see, it's overwriting the name variable! Yeah, I'll fix that.